packages/diagrams/backend/sirius-components-diagrams-layout/src/main/java/org/eclipse/sirius/components/diagrams/layout/incremental/LayoutStrategyEngineRegistry.java

```diff
@@ -0,0 +1,41 @@
+/*******************************************************************************
+ * Copyright (c) 2022 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.sirius.components.diagrams.layout.incremental;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import org.springframework.stereotype.Service;
+
+/**
+ * The registry containing all the {@link ILayoutStrategyEngine}.
+ *
+ * @author gcoutable
+ */
+@Service
+public class LayoutStrategyEngineRegistry implements ILayoutStrategyEngineRegistry {
+
+    private final List<ILayoutStrategyEngine> layoutStrategyEngines = new ArrayList<>();
+
+    @Override
+    public void add(ILayoutStrategyEngine layoutStrategyEngine) {
+        this.layoutStrategyEngines.add(layoutStrategyEngine);
+    }
+
+    @Override
+    public List<ILayoutStrategyEngine> getLayoutStrategyEngines() {
+        return Collections.unmodifiableList(this.layoutStrategyEngines);
+    }
+
+}
```

Comment:
As suspected, you can remove both registries and everything will be fine.

Reply:
Yes, both registries have been removed